### ISSUE TITLE
Plugin disabled at booting: java.lang.IllegalStateException: zip file closed

### ISSUE THREAD

**Issue by joex92** [xXastaspastaXx/Dimensions#3]
Hey! I love the plugin, It works great!

but recently it throws some errors at boot:
```
[Thu, 27. May 2021 17:53:05 UTC INFO] Enabling Dimensions v1.2.0
[Thu, 27. May 2021 17:53:05 UTC WARN] Enabled plugin with unregistered PluginClassLoader Dimensions v1.2.0
[Thu, 27. May 2021 17:53:05 UTC INFO] [DimensionsDebugger] There was an error with Dimensions. The plugin has been disabled. More info:
[Thu, 27. May 2021 17:53:05 UTC WARN] java.lang.IllegalStateException: zip file closed
[Thu, 27. May 2021 17:53:05 UTC WARN] at java.base/java.util.zip.ZipFile.ensureOpen(ZipFile.java:915)
[Thu, 27. May 2021 17:53:05 UTC WARN] at java.base/java.util.zip.ZipFile.getEntry(ZipFile.java:347)
[Thu, 27. May 2021 17:53:05 UTC WARN] at java.base/java.util.zip.ZipFile$1.getEntry(ZipFile.java:1121)
[Thu, 27. May 2021 17:53:05 UTC WARN] at java.base/java.util.jar.JarFile.getEntry0(JarFile.java:578)
[Thu, 27. May 2021 17:53:05 UTC WARN] at java.base/java.util.jar.JarFile.getEntry(JarFile.java:508)
[Thu, 27. May 2021 17:53:05 UTC WARN] at java.base/java.util.jar.JarFile.getJarEntry(JarFile.java:470)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:166)
[Thu, 27. May 2021 17:53:05 UTC WARN] at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:120)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:114)
[Thu, 27. May 2021 17:53:05 UTC WARN] at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
[Thu, 27. May 2021 17:53:05 UTC WARN] at me.xxastaspastaxx.dimensions.Dimensions.onEnable(Dimensions.java:47)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500)
[Thu, 27. May 2021 17:53:05 UTC WARN] at com.rylinaux.plugman.util.PluginUtil.enable(PluginUtil.java:63)
[Thu, 27. May 2021 17:53:05 UTC WARN] at com.rylinaux.plugman.command.EnableCommand.execute(EnableCommand.java:127)
[Thu, 27. May 2021 17:53:05 UTC WARN] at com.rylinaux.plugman.PlugManCommandHandler.onCommand(PlugManCommandHandler.java:95)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:172)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:827)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:776)
[Thu, 27. May 2021 17:53:05 UTC WARN] at us.ajg0702.startcommands.Main$1.run(Main.java:41)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:100)
[Thu, 27. May 2021 17:53:05 UTC WARN] at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:469)
[Thu, 27. May 2021 17:53:05 UTC WARN] at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1517)
[Thu, 27. May 2021 17:53:05 UTC WARN] at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:461)
[Thu, 27. May 2021 17:53:05 UTC WARN] at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1426)
[Thu, 27. May 2021 17:53:05 UTC WARN] at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1147)
[Thu, 27. May 2021 17:53:05 UTC WARN] at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:292)
```

I use [Plugman](https://www.spigotmc.org/resources/plugman-improved.88135/) to manage plugins, so once the server is fully booted I try to enable Dimensions manually and, oddly enough, it works! no problems... So the workaround I did was to use [ajStartCommands](https://www.spigotmc.org/resources/ajstartcommands.31033/) to automatically enable Dimensions after the server finish booting...

**Comment by xXastaspastaXx** [comment]
Do you have the ability to load the plugin only in server load and not by using any other means? If so please try that. If the errors is still there please let me now with the new error message

**Comment by joex92** [comment]
> Do you have the ability to load the plugin only in server load and not by using any other means? If so please try that. If the errors is still there please let me now with the new error message

ok... this is what is happening... I've been a bit busy and also trying some thing... so I've been using Yatopia  for more than half a year, and haven't got any issues... then on Yatopia 1.16.5 (build 11X) it stopped working at boot with this error:

```
13.06 07:09:19 [Server] INFO Enabling Dimensions v1.2.0
13.06 07:09:19 [Server] INFO Loaded 1 portals
13.06 07:09:21 [Server] INFO There was an error with Dimensions. The plugin has been disabled. More info:
13.06 07:09:21 [Server] WARN java.lang.NullPointerException
13.06 07:09:21 [Server] WARN at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:770)
13.06 07:09:21 [Server] WARN at com.sk89q.worldguard.WorldGuard.getPlatform(WorldGuard.java:100)
13.06 07:09:21 [Server] WARN at me.xxastaspastaxx.dimensions.portal.WorldGuardFlags.enablePlatform(WorldGuardFlags.java:71)
13.06 07:09:21 [Server] WARN at me.xxastaspastaxx.dimensions.Dimensions.onEnable(Dimensions.java:43)
13.06 07:09:21 [Server] WARN at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
13.06 07:09:21 [Server] WARN at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
13.06 07:09:21 [Server] WARN at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500)
13.06 07:09:21 [Server] WARN at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:518)
13.06 07:09:21 [Server] WARN at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:432)
13.06 07:09:21 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:594)
13.06 07:09:21 [Server] WARN at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:298)
13.06 07:09:21 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1069)
13.06 07:09:21 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:291)
13.06 07:09:21 [Server] WARN at java.base/java.lang.Thread.run(Thread.java:831)
13.06 07:09:21 [Server] INFO Disabling Dimensions v1.2.0```

then tried to reload it with a Plugin Manager plugin (Plugman in this case) and got this error since Yatopia 1.16.5 (Build 116):
```
13.06 07:10:12 [Server] INFO Enabling Dimensions v1.2.0
13.06 07:10:12 [Server] WARN Enabled plugin with unregistered PluginClassLoader Dimensions v1.2.0
13.06 07:10:12 [Server] INFO Loaded 1 portals
13.06 07:10:12 [Server] INFO There was an error with Dimensions. The plugin has been disabled. More info:
13.06 07:10:12 [Server] WARN java.lang.IllegalStateException: zip file closed
13.06 07:10:12 [Server] WARN at java.base/java.util.zip.ZipFile.ensureOpen(ZipFile.java:828)
13.06 07:10:12 [Server] WARN at java.base/java.util.zip.ZipFile.getEntry(ZipFile.java:327)
13.06 07:10:12 [Server] WARN at java.base/java.util.jar.JarFile.getEntry(JarFile.java:513)
13.06 07:10:12 [Server] WARN at java.base/java.util.jar.JarFile.getJarEntry(JarFile.java:468)
13.06 07:10:12 [Server] WARN at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:167)
13.06 07:10:12 [Server] WARN at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:586)
13.06 07:10:12 [Server] WARN at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:119)
13.06 07:10:12 [Server] WARN at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:114)
13.06 07:10:12 [Server] WARN at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:519)
13.06 07:10:12 [Server] WARN at me.xxastaspastaxx.dimensions.Dimensions.onEnable(Dimensions.java:47)
13.06 07:10:12 [Server] WARN at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
13.06 07:10:12 [Server] WARN at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
13.06 07:10:12 [Server] WARN at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500)
13.06 07:10:12 [Server] WARN at com.rylinaux.plugman.util.PluginUtil.enable(PluginUtil.java:63)
13.06 07:10:12 [Server] WARN at com.rylinaux.plugman.command.EnableCommand.execute(EnableCommand.java:127)
13.06 07:10:12 [Server] WARN at com.rylinaux.plugman.PlugManCommandHandler.onCommand(PlugManCommandHandler.java:95)
13.06 07:10:12 [Server] WARN at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
13.06 07:10:12 [Server] WARN at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
13.06 07:10:12 [Server] WARN at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:826)
13.06 07:10:12 [Server] WARN at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:776)
13.06 07:10:12 [Server] WARN at us.ajg0702.startcommands.Main$1.run(Main.java:41)
13.06 07:10:12 [Server] WARN at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:100)
13.06 07:10:12 [Server] WARN at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468)
13.06 07:10:12 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1427)
13.06 07:10:12 [Server] WARN at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:436)
13.06 07:10:12 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1342)
13.06 07:10:12 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1130)
13.06 07:10:12 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:291)
13.06 07:10:12 [Server] WARN at java.base/java.lang.Thread.run(Thread.java:831)
13.06 07:10:12 [Server] INFO Disabling Dimensions v1.2.0
13.06 07:10:12 [Server] INFO Dimensions has been enabled.```
so I went back to Yatopia 1.16.5 (Build 115) and the "reload after booting" option worked, so I'm there using than Build...

At the beginning I didn't pay attention to the ERRORS until, knowing that I'm using a server fork that almost all plugin devs believe is cursed, I went to ask for support on Yatopia's support. I realized that both errors were different so they told me it was a problem with WorldGuard, but it doesn't make sense, because some builds ago it was working fine with no errors whatsoever... so I tried with latest Purpur fork (1169), and finally I tried with latest PaperMC fork (777), and got the same results, it didn´t load on boot neither reloading after server finished booting... so now I'm here finally answering you... 

I have a guess that maybe since your latest release WorldGuard might have updated something... Just in case I have WorldGuard and WorldGuardExtraFlags... idk... I hope this info helps...

**Comment by xXastaspastaXx** [comment]
If the server has world guard installed then Dimensions cannot be reloaded because when the plugin loads it tries to register new world guard flags but it's not allowed unless it's on server startup. That's why I do not recommend using plugin managers to load/unload the plugin and instead use "/dim reload". Unfortunately there are not any known workarounds to fix this issue. That's why I asked if it was possible to load the plugin on server load only and not reload it.
Also there is no option to disable world guard support and I will not be updating any quick fix since I am currently re-writing Dimensions and I don't want to mess with the old code.
I am sorry for the inconvenience.


TL;DR
You must not use any plugin managers to reload Dimensions because world guard support breaks. Please restart your server and only use "/dim reload" in the future

**Comment by joex92** [comment]
> If the server has world guard installed then Dimensions cannot be reloaded because when the plugin loads it tries to register new world guard flags but it's not allowed unless it's on server startup. That's why I do not recommend using plugin managers to load/unload the plugin and instead use "/dim reload". Unfortunately there are not any known workarounds to fix this issue. That's why I asked if it was possible to load the plugin on server load only and not reload it.
> Also there is no option to disable world guard support and I will not be updating any quick fix since I am currently re-writing Dimensions and I don't want to mess with the old code.
> I am sorry for the inconvenience.
> 
> TL;DR
> You must not use any plugin managers to reload Dimensions because world guard support breaks. Please restart your server and only use "/dim reload" in the future

ok, I'll try with `/dim reload`, but why is the ERROR **at boot** happening?

**Comment by joex92** [comment]
ok now I´m at latest server build and still got the error at boot:
```
13.06 16:51:54 [Server] INFO Loaded 1 portals
13.06 16:51:54 [Server] INFO There was an error with Dimensions. The plugin has been disabled. More info:
13.06 16:51:54 [Server] WARN java.lang.NullPointerException
13.06 16:51:54 [Server] WARN at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:770)
13.06 16:51:54 [Server] WARN at com.sk89q.worldguard.WorldGuard.getPlatform(WorldGuard.java:100)
13.06 16:51:54 [Server] WARN at me.xxastaspastaxx.dimensions.portal.WorldGuardFlags.enablePlatform(WorldGuardFlags.java:71)
13.06 16:51:54 [Server] WARN at me.xxastaspastaxx.dimensions.Dimensions.onEnable(Dimensions.java:43)
13.06 16:51:54 [Server] WARN at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
13.06 16:51:54 [Server] WARN at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
13.06 16:51:54 [Server] WARN at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500)
13.06 16:51:54 [Server] WARN at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:518)
13.06 16:51:54 [Server] WARN at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:432)
13.06 16:51:54 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:596)
13.06 16:51:54 [Server] WARN at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:321)
13.06 16:51:54 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1078)
13.06 16:51:54 [Server] WARN at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:293)
13.06 16:51:54 [Server] WARN at java.base/java.lang.Thread.run(Thread.java:831)
13.06 16:51:54 [Server] INFO Disabling Dimensions v1.2.0
```
and doing `/dim reload` didn't work either:
```
13.06 16:52:24 [Server] INFO org.bukkit.command.CommandException: Cannot execute command 'dim' in plugin Dimensions v1.2.0 - plugin is disabled.
13.06 16:52:24 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:37) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:172) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:827) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:776) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at us.ajg0702.startcommands.Main$1.run(Main.java:41) ~[?:?]
13.06 16:52:24 [Server] INFO at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:100) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:469) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1518) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:461) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1427) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1148) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:293) ~[patched_1.16.5.jar:git-Yatopia-ver/1.16.5-120]
13.06 16:52:24 [Server] INFO at java.lang.Thread.run(Thread.java:831) [?:?]
```

**Comment by xXastaspastaXx** [comment]
Ok I am sorry, I never read that this error was on boot. I just assumed it happened because you attempted to reload the plugin using plugman.
The new errors you get are because the plugin is disabled (don't pay attention to them)

About the error you get when you boot your server:
It may happen because World guard is outdated OR updated (I will check sometime tomorrow and inform you)
It also may happen because the world guard is not enabling correctly or there is something interfering with it. Try to remove the extraworldguardflags plugin and check if this keeps happening. Also keep a backup of your worldGuars folder and delete it to make sure there is not any option cause problems.

**Comment by joex92** [comment]
went back to the build where it was working... and now is not working anymore... well I´ll let it disable itself... I hope you release the re-writting soon...

**Comment by joex92** [comment]
> Ok I am sorry, I never read that this error was on boot. I just assumed it happened because you attempted to reload the plugin using plugman.
> The new errors you get are because the plugin is disabled (don't pay attention to them)
> 
> About the error you get when you boot your server:
> It may happen because World guard is outdated OR updated (I will check sometime tomorrow and inform you)
> It also may happen because the world guard is not enabling correctly or there is something interfering with it. Try to remove the extraworldguardflags plugin and check if this keeps happening. Also keep a backup of your worldGuars folder and delete it to make sure there is not any option cause problems.

ok... I´ll try that... although I am using some of the ExtraFlags... I always keep WG updated...

**Comment by joex92** [comment]
Loaded without ExtraFlags... still NPE...

```
[23:06:59] [Server thread/INFO]: [Dimensions] Enabling Dimensions v1.2.0
[23:06:59] [Server thread/INFO]: [DimensionsDebugger] Loaded 1 portals
[23:06:59] [Server thread/INFO]: [DimensionsDebugger] There was an error with Dimensions. The plugin has been disabled. More info:
[23:06:59] [Server thread/WARN]: java.lang.NullPointerException
[23:06:59] [Server thread/WARN]: 	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:770)
[23:06:59] [Server thread/WARN]: 	at com.sk89q.worldguard.WorldGuard.getPlatform(WorldGuard.java:100)
[23:06:59] [Server thread/WARN]: 	at me.xxastaspastaxx.dimensions.portal.WorldGuardFlags.enablePlatform(WorldGuardFlags.java:71)
[23:06:59] [Server thread/WARN]: 	at me.xxastaspastaxx.dimensions.Dimensions.onEnable(Dimensions.java:43)
[23:06:59] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[23:06:59] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
[23:06:59] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500)
[23:06:59] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:518)
[23:06:59] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:432)
[23:06:59] [Server thread/WARN]: 	at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:596)
[23:06:59] [Server thread/WARN]: 	at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:321)
[23:06:59] [Server thread/WARN]: 	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1078)
[23:06:59] [Server thread/WARN]: 	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:293)
[23:06:59] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:831)
[23:06:59] [Server thread/INFO]: [Dimensions] Disabling Dimensions v1.2.0
```

**Comment by xXastaspastaXx** [comment]
Hm, can you please send me a list of your plugins? Also can you send me the world guard jar you are using (drag and drop here from your server, dont send me a link).

**Comment by joex92** [comment]
> Hm, can you please send me a list of your plugins? Also can you send me the world guard jar you are using (drag and drop here from your server, dont send me a link).

Sorry for the very late response... I've been very busy and we had some troubles with the hosting... GitHub doesn't allow me to upload `.jar` files, so I downloaded it and compressed it to zip: 
[worldguard-bukkit-7.0.5-dist.jar.zip](https://github.com/xXastaspastaXx/Dimensions/files/6796417/worldguard-bukkit-7.0.5-dist.jar.zip)

Plugins (152): AdvancedWeapons (1.5.1), ajParkour (2.12.2), ajStartCommands (2.0.3), ArbitraryPortals (1.0.0), AreaShop (2.6.3#CUSTOM), ArmedArmorStands (1.2.3), ArmorPlusPlus (0.6.5), Autorank (4.5.1), BackpacksRemastered (2020.0.5), BetterBeacons (2.3.1), BetterConcrete (1.7.0), BetterDoors (1.3), BetterEnd (4.4.5), BetterGiants (2.3), BetterSleeping3 (3.0.6), BiomeViruses (2.0.1), BKCommonLib (1.16.5-v3), BloodMoon (0.8), Bookshelf (3.0.3.0), BottledExp (3.0.0.1), BreakingSlab (1.0.1), BroadcastPlus (2.7.0), Carz (7.2), Chainmail (1.2.0), ChunkMiner (0.3), Citizens (2.0.27-SNAPSHOT (build 2191)), ClearLag (3.2.2), CMILib (1.0.2.10), Coins (1.10.5), CommandSigns (2.2.0), CraftableNotch (1.0), CraftBook (3.10.3;4680-f1ee2df), CustomTime (2.0), DailyRewards (1.0.7), DamageManager (1.0.1), DangerousCaves (2.2.13;24), DeadSouls (1.6), Denizen (1.2.0-SNAPSHOT (build 1739-REL)), DieMusic (1.0-SNAPSHOT), DiscordSRV (1.23.0), DiscordSRV-Staff-Chat (1.3.4), DiscordSRVUtils (1.2.0), Drop2Inventory (4.9.5), Drycell (1.0.7), DualWield (2.2), dynmap (3.2-beta-2-483), dynmap-structures (2.3.1), Dynmap-WorldGuard (1.2), DynmapBannerMarkers (1.1.0), DynmapEssentials (1.1), DynMapNPC (2.0.0), EconomyShopGUI (3.3), Elevator (3.11.1), elyby-skinsystem (0.2.0-beta), EnchantmentsExtractor (1.4), EnchantPlus (1.3.5), Enchantssquared (1.5), Endermaning (1.1.4), EnhancedEnchants (1.13_1.2), EpicPluginLib (2.0), Essentials (2.19.0-dev+218-cca98e8), EssentialsChat (2.19.0-dev+218-cca98e8), EssentialsGeoIP (2.19.0-dev+218-cca98e8), EssentialsSpawn (2.19.0-dev+218-cca98e8), ExtraContexts (2.0-SNAPSHOT), FarmControl (1.1.8), FastAsyncWorldEdit (1.16-735;b0c0689), Gaea (1.15.0), Geyser-Spigot (1.4.0-SNAPSHOT), GodItems (1.6.1), GoldenChallenges (1.0.7), GriefPrevention (16.17.1-25-g4f95938), GriefPreventionDynmap (1.0), GSit ([5.3.8.1]), HackedClientDetect (1.0.2), HeadDB (2.3.1), HealthBar (1.2), HolographicDisplays (2.4.7), HoloMobHealth (2.2.2.0), Images (2.2.1), InfiniteVouchers (2.2.0), InteractionVisualizer (1.16.2.2), InteractiveChat (4.0.1.0), InvisibleItemFrames (1.2.1), Jobs (5.0.0.2), LagMonitor (1.17.2-SNAPSHOT-5c40f7f), LightAPI (3.5.1), LockettePro (2.10.7), LuckPerms (5.3.12), mcMMO (2.1.200-SNAPSHOT), MineChess (1.12.1), MobsToEggs (1.5.5_5), MoreArmour (2.0.2), MoreMobHeads (1.15_1.0.21), Multiverse-Core (4.3.0-b846), Multiverse-EssentialsSupport (2.5-b3), Multiverse-Inventories (4.2.2-SNAPSHOT-bUNKNOWN), Multiverse-NetherPortals (4.2.2-SNAPSHOT-b801), Multiverse-Portals (4.2.2-SNAPSHOT-b841), MultiverseHardcore (1.0.3), MusicBox (2.0.4), MyCommand (5.7.1), NexEngine (2.0.1), NoPhysics (1.4), NoteBlockAPI (1.6.1-SNAPSHOT), NotTooExpensive (1.2.4), Oh_the_dungeons_youll_go (15.3), Orebfuscator (5.1.3), PerWorldChat (0.0.2), PlaceholderAPI (2.10.10), PlugMan (2.2.2), PortalGun (2.1), ProtocolLib (4.7.0), RageMode (1.8.8), RandomTeleport (2.0.0.68), RealEstate (1.2.2), RealScoreboard (05-11-2020 Build 1), Relics (2.2), RocketJoin (2.0), SecurityLog (1.0), Shopkeepers (2.12.0), ShulkerPacks (1.5.11), SilkSpawners (7.1.0), SmoothTimber (1.16.5), StayPut (1.2.1), StructuresCraftable (1.0-SNAPSHOT), Tamablefoxes (2.1.1-SNAPSHOT), TangledMaze (1.4.2), TerraformGenerator (4.6.2), TreysDoubleJump (2.6.5), TrollingFreedom (2.3.5), UltimateRevenge (7.6.1), Uncraftables (1.1.0), Vault (1.7.3-b131), ViaBackwards (4.0.1), ViaRewind (2.0.1), ViaVersion (4.0.1), VillagerInventory (2.0), VillagerOptimiser (1.1.5), VoidGenerator (1.7.0), VoidTeleport (2.3.7), WebConsole (2.1), Webhost (1.09), WG-GUI (1.5.3), WhitelistPro (1.1), WorldEditSelectionVisualizer (2.1.0), WorldGuard (7.0.5+3827266), WorldGuardExtraFlags (4.1.5-SNAPSHOT), ZeusFriends (1.0), ZombieApocalypse (1.4.2), Dimensions (1.2.0), IdleBot (1.2.0)
